lib/jsonapi/utils/underscore.ex

```diff
@@ -3,7 +3,7 @@ defmodule JSONAPI.Utils.Underscore do
   Helpers for replacing underscores with dashes.
   """
 
-  def underscore?, do: Application.get_env(:jsonapi, :underscore_to_dash, false)
+  def underscore?, do: Application.get_env(:jsonapi, :underscore_to_dash, false) != false
```

Comment:
Should we make the config a module attribute above and use it here and down in the `underscore?/1`?

Reply:
Sure—good idea

Reply:
Well, I guess not—the module attribute doesn't work for our tests, since we change the config at runtime—I made the config into a function instead